### DISCUSSION TITLE
How to determine if two words are derived from the same word?

### DISCUSSION THREAD

**Discussion by rbracco** [explosion/spaCy#11452] [Help: Coding & Implementations]
I would like to be able to check if two words are derived from the same word (e.g. monster/monstrous). In this case, since they are different parts of speech (noun/adj), they have different lemmas. Is there a way to do this in spaCy?

Sorry, the answer to this probably exists but I'm not sure how to word it so I haven't been able to find anything except https://github.com/explosion/spaCy/issues/327 which is from 2016 and suggests using wordnet, which I'll do if it's not possible in spaCy in 2022.

**Comment by polm** [comment]
spaCy doesn't have a feature to detect derivationally related words, and it still doesn't have a stemmer (or plans to add one). 

I think that the issue you'll run into is that "derivationally related forms" are often unclear and not easily defined, which is why you'd need to pick a particular database (like WordNet) or use an approximation (like stemming). 

Can you describe what you want to use this for? Usually features like this aren't very useful to modern models, or similar information can be captured with more general subword features.
